pkg/monitor/intervalcreation/node_test.go

```diff
@@ -253,13 +262,39 @@ func TestMonitorApiIntervals(t *testing.T) {
 				To:   systemdJournalLogTime("Feb 01 05:37:45.731611"),
 			},
 		},
+		{
+			name:          "too many netlink events",
+			logLine:       `Apr 12 11:49:49.188086 ci-op-xs3rnrtc-2d4c7-4mhm7-worker-b-dwc7w NetworkManager[1155]: <info> [1681300187.8326] platform-linux: netlink[rtnl]: read: too many netlink events. Need to resynchronize platform cache`,
+			generatorFunc: intervalsFromNetworkManagerLogs,
+			want: monitorapi.Interval{
+				Condition: monitorapi.Condition{
+					Level:   monitorapi.Warning,
+					Locator: "node/ci-op-xs3rnrtc-2d4c7-4mhm7-worker-b-dwc7w",
+					Message: "[1155]: <info> [1681300187.8326] platform-linux: netlink[rtnl]: read: too many netlink events. Need to resynchronize platform cache",
+					StructuredLocator: monitorapi.Locator{
+						Type: monitorapi.LocatorTypeNode,
+						Keys: map[monitorapi.LocatorKey]string{
+							"node": "testName",
+						},
+					},
+					StructuredMessage: monitorapi.Message{
+						Reason:       "",
+						Cause:        "",
+						HumanMessage: "NetworkManager[1155]: <info> [1681300187.8326] platform-linux: netlink[rtnl]: read: too many netlink events. Need to resynchronize platform cache",
+						Annotations:  map[monitorapi.AnnotationKey]string{},
+					},
+				},
+				From: systemdJournalLogTime("Apr 12 11:49:49.188086"),
+				To:   systemdJournalLogTime("Apr 12 11:49:49.188086"),
```

Comment:
For the unit test failure, I would expect with the recent code change it was telling us it should be 11:49:50 for the To Date but is this instead saying no To value has been set?

             	            	expected: time.Date(2023, time.April, 12, 11, 49, 49, 188086000, time.UTC)
            	            	actual  : time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC)

Reply:
I might have missed this in a previous review  https://github.com/dgoodwin/origin/blob/3d2edd51aaf2279f2810e33289a4125ab096dc87/pkg/monitor/monitorapi/construction.go#L59